sdmetrics/timeseries/base.py

```diff
@@ -61,6 +63,14 @@ def _validate_inputs(cls, real_data, synthetic_data, metadata=None, sequence_key
                 if field not in real_data.columns:
                     raise ValueError(f'Field {field} not found in data')
 
+            for column, sdtype in metadata['columns'].items():
+                if sdtype['sdtype'] == 'datetime':
+                    try:
+                        real_data[column] = pd.to_datetime(real_data[column])
```

Comment:
Does it make sense to look for the datetime format in sdtype dict and pass it to pd.to_datetime (if present)?